modin/engines/base/io/column_stores/parquet_reader.py

```diff
@@ -66,6 +66,17 @@ def _read(cls, path, engine, columns, **kwargs):
                 pd = ParquetDataset(path)
                 meta = pd.metadata
                 column_names = pd.schema.names
+            elif (
+                isinstance(path, str)
+                and "://" in path
+                and not path.startswith(("http://", "https://"))
```

Comment:
Is http:// or https:// handled by `ParuqetFile` constructor? The documentation did not say: https://arrow.apache.org/docs/python/generated/pyarrow.parquet.ParquetFile.html

Reply:
You're right. It was fixed. Now I try to handle `hdfs://` beginning only.